core/arch/x86/opcode.h

```diff
@@ -1266,6 +1266,59 @@ enum {
     /* 1105 */ OP_adox, /**< IA-32/AMD64 adox opcode. */
     /* 1106 */ OP_adcx, /**< IA-32/AMD64 adox opcode. */
 
+    /* Intel AVX-512 */
+    /* 1107 */ OP_kmovw,    /**< IA-32/AMD64 AVX-512 kmovw opcode */
```

Comment:
All the existing doxygen comments in a period.  Best to match them for consistency, to match the style guide, etc.

Reply:
Done (will update).